### ISSUE TITLE
How to use masks in T5

### ISSUE THREAD

**Issue by puraminy** [ThilinaRajapakse/simpletransformers#1062]
How can I do the following T5 code in Huggingface, with Simpletransformers?

```
input_ids = tokenizer('The <extra_id_0> walks in <extra_id_1> park', return_tensors='pt').input_ids
labels = tokenizer('<extra_id_0> cute dog <extra_id_1> the <extra_id_2>', return_tensors='pt').input_ids
# the forward function automatically creates the correct decoder_input_ids
loss = model(input_ids=input_ids, labels=labels).loss
```

from:
https://huggingface.co/transformers/model_doc/t5.html

**Comment by ThilinaRajapakse** [comment]
This is not currently supported in Simple Transformers.

**Comment by Liujingxiu23** [comment]
@stale hi  I did not understand, why no only "attention_mask"， I suppose there are two mask， encoder_attention_mask and decoder_attention_mask， why there is only one mask in the code of T5?
